src/Iris.php

```diff
@@ -54,16 +53,32 @@ public function __construct()
         add_action('plugins_loaded', [__CLASS__, 'check_for_offload_media']);
 
         // IRIS :)
-        add_filter('wp_generate_attachment_metadata', [__CLASS__, 'iris_webp_converter'], 10, 3);
+        add_filter('wp_generate_attachment_metadata', [__CLASS__, 'irisWebpConverter'], 10, 3);
+        add_filter('wp_generate_attachment_metadata', [__CLASS__, 'iris'], 10, 3);
         add_action('delete_attachment', [__CLASS__, 'delete_webp_conversions'], 10);
```

Comment:
Code style: could you update this to follow the same casing structure as the rest of the class methods `deleteWebpConversions`?